frontend/test/pytest/test_mitigation.py

```diff
@@ -308,5 +315,30 @@ def mitigated_qnode():
     assert np.allclose(mitigated_qnode(), circuit())
 
 
+def test_exponential_extrap_with_kwargs():
```

Comment:
micro-nit: may as well just write the full word 😀

`extrap` sounds like a musical genre that has waned in popularity.

Reply:
fixed (along with a few other instances) here: https://github.com/PennyLaneAI/catalyst/pull/953/commits/eea7d4d27408b4351525acd4ea39310841bb9e46